src/goose/brokers/redis/api/dead_jobs.clj

```diff
@@ -21,24 +21,41 @@
         match? (fn [job] (= (:id job) id))]
     (first (find-by-pattern redis-conn match? limit))))
 
+;; TODO: Return job instead of redis txn
 (defn replay-job [redis-conn job]
   (let [sorted-set d/prefixed-dead-queue]
-    (when (redis-cmds/sorted-set-score redis-conn sorted-set job)
+    (when (not-every? nil? (redis-cmds/sorted-set-scores redis-conn sorted-set job))
```

Comment:
question: What has changed here? Shouldn't this function be unchanged since console uses the newer function?

Reply:
This is a different change.
I have refactored `sorted-set-score` to sorted-set-scores` to take >=1 job/s at redis commands layer.